src/main/java/com/avispl/symphony/dal/communicator/aver/ptz/AverPTZCommunicator.java

```diff
@@ -183,7 +204,175 @@ public void initAverRestCommunicator() throws Exception {
 	 */
 	@Override
 	public void controlProperty(ControllableProperty controllableProperty) {
+		String property = controllableProperty.getProperty();
+		String value = String.valueOf(controllableProperty.getValue());
+		controlOperationsLock.lock();
+
+		try {
+			// Power
+			if (property.startsWith(Command.POWER.getName())) {
+				if (value.equals("1")) {
+					powerOn();
+				} else if (value.equals("0")) {
+					powerOff();
+				}
+			}
+
+			// Zoom
+			else if (property.startsWith(Command.ZOOM.getName())) {
+				if (property.endsWith(ZoomControl.TELE.getName())) {
+					zoomTele();
+				} else if (property.endsWith(ZoomControl.WIDE.getName())) {
+					zoomWide();
+				}
+			}
+
+			// Focus
+			else if (property.startsWith(Command.FOCUS.getName())) {
+				if (property.endsWith(Command.FOCUS_MODE.getName())) {
+					if (value.equals("1")) {
```

Comment:
Please move "1" and "0" to constant. Thx!

Reply:
ok thank you, I will do it